virttest/qemu_vm.py

```diff
@@ -4315,6 +4315,36 @@ def check_block_locked(self, value):
                     return block['locked']
         return False
 
+    def check_device_tray_opened(self, value):
```

Comment:
IMHO, when this function returns `True` or `False` at the end, it may be more clear to naming it `is_device_tray_opened`.
`check_device_tray_opened` can be used for a complete step or several steps that can result to test fail or error.